pwnlib/util/misc.py

```diff
@@ -151,7 +152,9 @@ def which(name, all = False):
     if os.path.sep in name:
         return name
 
-    isroot = os.getuid() == 0
+    if sys.platform != 'win32':
```

Comment:
This could just be changed to:

```py
isroot = False if sys.platform == 'win32' else (os.getuid() == 0)
```

The other changes in this file would then be unnecessary.